packages/react-sdk/src/remix.ts

```diff
@@ -0,0 +1,47 @@
+const getRemixSegment = (segment: string) => {
+  if (segment === "*") {
+    return "$";
+  }
+  const match = segment.match(/^:(?<name>\w+)(?<modifier>\*|\?)?$/);
+  const name = match?.groups?.name;
+  const modifier = match?.groups?.modifier;
+  if (name) {
+    if (modifier === "*") {
+      return "$";
+    }
+    if (modifier === "?") {
+      return `($${name})`;
```

Comment:
https://remix.run/docs/en/main/file-conventions/routes#escaping-special-characters
Interesting that in remix spec special chars should be inside `[`, `]` and nothing is written about `(` and `)`
Imo should be tested with various chars

Reply:
`($lang)` is optional segment